core/src/main/resources/org/apache/spark/ui/static/streaming-page.css

```diff
@@ -31,7 +31,7 @@
 }
 
 .tooltip-inner {
-  max-width: 500px !important; // Make sure we only have one line tooltip
```

Comment:
I just found this incorrect comment as per the error log. So I think we should not totally turn off all logs. That's why this PR only ignores logs from "bootstrap.min.css" and "vis.min.css".